src/models/users/user-models.test.js

```diff
@@ -0,0 +1,74 @@
+let userModel = require("./user-models.js");
+let eventModel = require("../events/event-models.js");
+const db = require("../../../data/dbConfig.js");
+
+const newUser = {
+  Email: String(Math.random()),
+  Password: "secret",
+  FirstName: "John",
+  LastName: "Doe",
+  Address: "Portland",
+  Gender: "male",
+  Latitude: 22.222,
+  Longitude: 1.22,
+};
+
+const updatedUser = {
+  Email: String(Math.random()),
+  Password: "moreSecret",
+  FirstName: "Jane",
+  LastName: "Doe",
+  Address: "Portland",
+  Gender: "female",
+  Latitude: 22.222,
+  Longitude: 5.55,
+};
+
+describe("user models", () => {
+  let createdUserId;
+  let userCount;
+
+  beforeAll(async () => {
+    const users = await db("Users");
+    userCount = users.length;
+  });
+
+  test("creates a user", async () => {
+    const added = await userModel.add(newUser);
+    createdUserId = added.id;
+    const users = await db("Users");
+    expect(users).toHaveLength(userCount + 1);
```

Comment:
Again, this seems like a good approach, but are we certain the tests are executed sequentially?